langs/en.json

```diff
@@ -208,7 +208,7 @@
     "SY": "Syrian Arab Republic",
     "TW": ["Taiwan, Province of China", "Taiwan"],
     "TJ": "Tajikistan",
-    "TZ": "Tanzania, United Republic of",
+    "TZ": ["Tanzania, United Republic of", "Tanzania"],
```

Comment:
While you are at it, could you change Tanzania, United Republic of to United Republic of Tanzania as well?